src/librustdoc/html/static/js/main.js

```diff
@@ -1219,6 +1219,112 @@ href="https://doc.rust-lang.org/${channel}/rustdoc/how-to-read-rustdoc.html\
     searchState.setup();
 }());
 
+(function() {
```

Comment:
We should have done that for other parts of this file too but can you add a comment saying that this function/part is to handle the sidebar resize please?